src/components/chat/chat.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import ForumIcon from '@material-ui/icons/Forum';
+import CancelIcon from '@material-ui/icons/Cancel';
+import MailOutlineIcon from '@material-ui/icons/MailOutline';
+import MessengerCustomerChat from 'react-messenger-customer-chat';
+import { useSelector } from 'react-redux';
+import { Button, TextField } from '@material-ui/core';
+import { useStyles } from './chat.style';
+import { CHAT } from '../../translations/chat.translation';
+import kyivstar from '../../images/kyivstar.png';
+import vodafone from '../../images/vodafone.png';
+
+export const Chat = () => {
+  const [visible, setBlock] = useState(false);
+  const [mailFormVisible, setMailFormVisible] = useState(false);
+  const { language, themeMode } = useSelector((state) => ({
+    language: state.Language.language,
+    themeMode: state.Theme.lightMode
+  }));
+  const style = useStyles({ themeMode, visible, mailFormVisible });
+
+  const activeMessenger = () => (
```

Comment:
Please, move this component to the separate file, it definitely looks like a separate component

Reply:
done